### PR TITLE
An example for atari game and play mode for visualization

### PR DESCRIPTION
**Pull request by emailweixu**
It takes about 2 hours to reach reward of 18 - 19

### CODE REVIEW

File: alf/algorithms/on_policy_algorithm.py [hunk 1]
```diff
@@ -175,16 +176,50 @@ def action_distribution_spec(self):
...
 
+    def greedy_predict(self, time_step: ActionTimeStep, state=None):
+        """Predict for one step of observation.
+
+        Generate greedy action that maximizes the action probablity).
+
+        Args:
+            time_step (ActionTimeStep):
+            state (nested Tensor): should be consistant with train_state_spec
+
```
Comment (hnyu) [inline alf/algorithms/on_policy_algorithm.py]: consistant -> consistent

File: alf/examples/actor_critic.py [hunk 2]
```diff
@@ -181,6 +194,13 @@ def train_eval(train_dir, debug_summaries=False):
         train_dir, env, algorithm, debug_summaries=debug_summaries)
 
 
+def play(train_dir):
+    """A simple train and eval for ActorCriticAlgorithm."""
```
Comment (hnyu) [inline alf/examples/actor_critic.py]: comment to be updated

File: alf/trainers/on_policy_trainer.py [hunk 3]
```diff
@@ -108,3 +108,38 @@ def train_():
...
+    if use_tf_functions:
+        driver.run = tf.function(driver.run)
+
+    env.reset()
+    time_step = driver.get_initial_time_step()
+    policy_state = driver.get_initial_state()
+    for _ in range(num_steps):
+        time_step, policy_state = driver.run(
+            max_num_steps=1, time_step=time_step, policy_state=policy_state)
+        env.pyenv.envs[0].render(mode='human')
```
Comment (hnyu) [inline alf/trainers/on_policy_trainer.py]: Maybe potentially give users an option to render in other modes? For example, mode='rgb_array' for saving a video of playing the game, which is helpful if the rendering is on a server.

File: alf/algorithms/on_policy_algorithm.py [hunk 4]
```diff
@@ -175,16 +176,50 @@ def action_distribution_spec(self):
     def predict(self, time_step: ActionTimeStep, state=None):
         """Predict for one step of observation.
 
+        Args:
+            time_step (ActionTimeStep):
+            state (nested Tensor): should be consistant with train_state_spec
```
Comment (hnyu) [inline alf/algorithms/on_policy_algorithm.py]: consistant -> consistent

File: alf/trainers/on_policy_trainer.py (hunk 3 above)
Comment (emailweixu) [reply alf/trainers/on_policy_trainer.py]: for mode 'rgb_array', we need to add code for saving the images. That can be done in a future change if it's useful.

File: alf/trainers/on_policy_trainer.py [hunk 5]
```diff
@@ -114,9 +115,23 @@ def train_():
 def play(train_dir,
          env,
          algorithm,
+         greedy_predict=True,
```
Comment (hnyu) [inline alf/trainers/on_policy_trainer.py]: Should this argument be passed to OnPolicyDriver() below?

**Pull request merged**